charts/argo-cd/templates/redis/service.yaml

```
@@ -11,7 +11,8 @@ metadata:
     {{- end }}
   {{- with .Values.redis.service.annotations }}
   annotations:
-    {{- toYaml . | nindent 4 }}
+  {{- range $key, $value := .Values.redis.service.annotations }}
```

Scope ist different here as we use `with`
```suggestion
  {{- range $key, $value := . }}
```